truss/protostuff/protostuff.go

```diff
@@ -0,0 +1,196 @@
+// Package protostuff provides an interface for interacting with protoc
+// requiring only paths to files on disk
+
+package protostuff
```

Comment:
`protogen` could be good. Many of these functions parse things rather than generate things. @hasIan  @hasLeland, `protogen`?